src/sync/sync_session.cpp

```diff
@@ -423,6 +423,7 @@ void SyncSession::handle_error(SyncError error)
             case ProtocolError::reuse_of_session_ident:
             case ProtocolError::bound_in_other_session:
             case ProtocolError::bad_message_order:
+            case ProtocolError::pong_timeout:
```

Comment:
Actually, I think you're right, that's probably not what we want. I've posted https://github.com/realm/realm-sync/pull/1187#issuecomment-290212966 to get some clarification. Thanks!